.github/workflows/build-jenkins-plugin-release.yml

```diff
@@ -0,0 +1,19 @@
+name: Build Jenkins Plugin
+
+on:
+  push:
+    tags:
+      - BIT*
```

Comment:
Why not just build all tags, and then create a tag like:
`bosch-bitbucket-branch-source-2.9.11.1`